08_podstawy_js_gra_kamien_nozyce_papier/.gitignore

```diff
@@ -73,4 +73,5 @@ typings/
 .serverless
 
 # CUSTOM
-sass/*.css
\ No newline at end of file
+sass/*.css
+css/*.css
```

Comment:
dlaczego dodajesz to do gitignore?

Reply:
Folder CSS: autoprefixer na Macu i Windowsie inaczej zapisywał wyjściowe pliki CSS, w związku z czym z każdym commitem powstawała nic nie wnosząca różnica. Pliki CSS zawsze można wygenerować taskiem buildującym. Ale OK, przywrócę śledzenie CSS bo faktycznie w tym projekcie nie ma tworzenia paczki jako takiej.

Folder SASS: pliki CSS musiały mi się pojawić przypadkowo w trakcie konfiguracji tasków i je roboczo zakomentowałem, wyrzucę tą linikję.